src/main/java/uk/gov/justice/hmpps/prison/service/curfews/OffenderCurfewService.java

```diff
@@ -124,31 +124,26 @@ private Set<Long> curfewBookingIds(Stream<OffenderCurfew> ocs) {
         return ocs.map(OffenderCurfew::getOffenderBookId).collect(toSet());
     }
 
-    @VerifyOffenderAccess(overrideRoles = {"SYSTEM_USER", "VIEW_PRISONER_DATA"})
```

Comment:
This annotation wasn't doing anything as it was VerifyBookingAccess that was needed